robottelo/ui/contentenv.py

```diff
@@ -24,16 +33,28 @@ def create(self, name, description=None, prior=None):
                 common_locators['description'], description)
         self.click(common_locators['create'])
 
+    def search(self, name):
```

Comment:
Nope, it is a corner case without search box or search button. Or even entities list :)